contrib/terraform/openstack/variables.tf

```diff
@@ -156,3 +156,19 @@ variable "worker_allowed_ports" {
     }
   ]
 }
+
+variable "use_loadbalancer" {
+  default = 0
```

Comment:
Would it make more sense to use a boolean here? https://www.terraform.io/docs/configuration/variables.html#booleans

Reply:
I actually used this variable to provide the "count" for the load balancer creation here: https://github.com/rcbops/kubespray/pull/3/files#diff-666fabfb9650ae57a1969cde9aaa575dR2.  I'm sure there's a better way to do it, open to suggestions.

Reply:
This is probably fine.